dask_sql/physical/rel/custom/predict.py

```diff
@@ -2,6 +2,15 @@
 import uuid
 from typing import TYPE_CHECKING
 
+import numpy as np
+
+try:
+    from dask_ml.wrappers import ParallelPostFit
```

Comment:
Depending on whether this PR or #832 is merged first, this chunk of code can be replaced with a `from dask_sql.physical.rel.custom.wrappers import ParallelPostFit`